source/features/command-palette-navigation-shortcuts.tsx

```diff
@@ -0,0 +1,32 @@
+import select from 'select-dom';
+import onetime from 'onetime';
+
+import features from '.';
+
+function commandPaletteKeydown({key, ctrlKey, currentTarget}: KeyboardEvent): void {
+	if (!currentTarget || !ctrlKey || (key !== 'n' && key !== 'p')) {
+		return;
+	}
+
+	const targetKey = key === 'n' ? 'ArrowDown' : 'ArrowUp';
+	currentTarget.dispatchEvent(new KeyboardEvent('keydown', {bubbles: true, key: targetKey, code: targetKey}));
```

Comment:
We probably need to call `preventDefault()` somewhere in the listener because of native browsers shortcuts (e.g. Ctrl+P opens up the print dialog in Firefox).

Reply:
I'll add `preventDefault()`. `ctrl-n/p` are working for me in Firefox on macOS, fwiw.